src/huggingface_hub/serialization/_dduf.py

```diff
@@ -0,0 +1,142 @@
+import logging
+import mmap
+import shutil
+import zipfile
+from contextlib import contextmanager
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import Dict, Generator, Union
+
+from ..errors import DDUFCorruptedFileError
+
+
+logger = logging.getLogger(__name__)
+
+DDUF_ALLOWED_ENTRIES = {".json", ".gguf", ".txt", ".safetensors"}
```

Comment:
Let's add ".model", ".bin", ".pt" as well? 

Most of the ecosystem libraries now enforce a `weights_only=True` when using `torch.load()` -- so the PyTorch pickle should probably be okay to add here. 

`.model` is for the sentence-piece tokenizer: 
https://huggingface.co/black-forest-labs/FLUX.1-dev/blob/main/tokenizer_2/spiece.model

Reply:
> ".bin", ".pt" as well?

i think we said we deliberately don't want to support legacy pickle otherwise the format isn't secure

Reply:
i think it's a good constraint to put

Reply:
Okay. .model is still needed.

Reply:
Done adding  `.model` to the list